include/odp/api/spec/packet_io_types.h

```diff
@@ -60,6 +60,12 @@ extern "C" {
  * Invalid LSO profile handle
  */
 
+/**
+ * @def ODP_PKTIO_MAX_INDEX
+ * Maximum packet IO interface index. Use odp_pktio_max_index() to check the
+ * runtime maximum value, which may be smaller than this define.
```

Comment:
"... than this define" -> "... than this" or "... than this value" would perhaps sound more natural.

Reply:
Changed to "than this value" in V2.